### ISSUE TITLE
Delete torrent shows snackbar error for synology nas

### ISSUE THREAD

**Issue by ndorigatti** [erickok/transdroid#300]
When trying to delete a torrent from a synology nas connection (download station), a snackbar saying error, check your settings is shown. It was working before synology DSM update so i think is a change in API. I'm using trandroid 2.5.6

**Comment by rvanpruissen** [comment]
Could this be related to a change in user management. I've seen other scripts having to change from admin to postgres.

**Comment by erickok** [comment]
Can you still see your torrents listing? Does the delete work (despite it showing the error)?

**Comment by rvanpruissen** [comment]
No, delete doesn't work. I'm using the download station app for this now. Tell me if you want me to test something.

**Comment by borovac** [comment]
Hey guys, I don't know what solved the issue, but it is working again for me. It connects, shows the torrents and I can finally delete them.

**Comment by PeterBerre** [comment]
borovac: same here. 
I don't know when it started working again, but i think one of the recent synology patches, or ds download updates fixed it.

There is another issue though. occasionally, for new downloads, the ratio only shows as NaN.
Oddly, if i remove/delete another torrent, the ratio is displayed correctly. 
I think this is a synology issue. I'm going to test it a bit more.
